examples/AspNet/Examples.AspNet.csproj

```diff
@@ -79,13 +79,13 @@
     <PackageReference Include="Microsoft.AspNet.WebApi.WebHost" Version="[5.2.7,6.0)" />
     <PackageReference Include="Microsoft.AspNet.Mvc" Version="[5.2.7,6.0)" />
     <PackageReference Include="Microsoft.AspNet.WebPages" Version="[3.2.7,4.0)" />
-    <PackageReference Include="OpenTelemetry" Version="1.3.0" />
+    <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
     <PackageReference Include="OpenTelemetry.Instrumentation.Http" Version="1.0.0-rc9.4" />
-    <PackageReference Include="OpenTelemetry.Exporter.Console" Version="1.3.0" />
-    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.3.0" />
-    <PackageReference Include="OpenTelemetry.Exporter.Jaeger" Version="1.3.0" />
-    <PackageReference Include="OpenTelemetry.Exporter.Zipkin" Version="1.3.0" />
-    <PackageReference Include="OpenTelemetry.Exporter.Prometheus" Version="1.3.0-rc.2" />
+    <PackageReference Include="OpenTelemetry.Exporter.Console" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="OpenTelemetry.Exporter.Jaeger" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="OpenTelemetry.Exporter.Zipkin" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="OpenTelemetry.Exporter.Prometheus.HttpListener" Version="1.4.0-beta.2" />
```

Comment:
which part is misleading? Upgrading prometheus.httplistener?

Reply:
prometheus do not have a stable release yet, so it has to be beta. is that the thing you are referrring to as well?

Reply:
I would use here OpenTelemetry.Exporter.Prometheus v. 1.3.0-rc.2 to avoid implicit upgrade OTel to 1.4.0-beta.2.

Even if it is not an _stable_ release, it does not bring anything from beta channel.

Reply:
got it. thanks for catching this.